### PR TITLE
LPD-16658 show published time message if file scheduled

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://liferay.atlassian.net/browse/LPD-16658

https://github.com/liferay-content-management/liferay-portal/assets/47524751/22206072-f064-4ffc-9d40-f014cf9421b5

- **LPD-16658 show published time message if file scheduled**
- **LPD-16658 add playwright test**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:playright

**Comment by AliciaGarciaGarcia** [comment]
ci:test:playwright

File: modules/test/playwright/tests/document-library-web/fileEntry.spec.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+test('LPD-16658 Show a success message after scheduling a new file', async ({
+	documentLibraryEditFilePage,
+	documentLibraryPage,
+	page,
+}) => {
+	const scheduleDate = `01/01/${new Date().getFullYear() + 1}`;
+	const title = getRandomString();
+
+	const fileEntry = page.getByRole('link', {name: title});
```
Comment (ambrinchaudhary) [inline modules/test/playwright/tests/document-library-web/fileEntry.spec.ts:30]: Inline this in the expect sentence

File: modules/test/playwright/pages/document-library-web/DocumentLibraryEditFilePage.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+				'false' ||
+			!(await this.scheduleButton.getAttribute('aria-expanded'));
+
+		if (isClosed) {
+			await this.scheduleButton.click();
+		}
+
+		await this.publishDateSelector.click();
+		await this.publishDateSelector.fill(scheduleDate);
+		await this.publishDateSelector.click();
```
Comment (ambrinchaudhary) [inline modules/test/playwright/pages/document-library-web/DocumentLibraryEditFilePage.ts:49]: Is this second `click()` really needed?

**Review by ambrinchaudhary — APPROVED** [review]
Just to minor comments, otherwise LGTM!

File: modules/test/playwright/pages/document-library-web/DocumentLibraryEditFilePage.ts (hunk 2 above)
Comment (AliciaGarciaGarcia) [reply modules/test/playwright/pages/document-library-web/DocumentLibraryEditFilePage.ts:49]: this is needed to close the calendar modal.

File: modules/test/playwright/tests/document-library-web/fileEntry.spec.ts (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/test/playwright/tests/document-library-web/fileEntry.spec.ts:30]: ok, I will push an additional commit to inline this and rename the test suite to testFeatureFlagsEnabled

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:playwright

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2549083115

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2549182449

**Comment by AliciaGarciaGarcia** [comment]
it seems that the test is failing because the calendar modal is not closed
https://testray.liferay.com/reports/production/logs/2024-03/test-1-37/test-portal-acceptance-pullrequest%28master%29/16295/playwright-js-tomcat90-mysql57-jdk8/20/0/playwright-report/trace/index.html?trace=https://testray.liferay.com/reports/production/logs/2024-03/test-1-37/test-portal-acceptance-pullrequest%28master%29/16295/playwright-js-tomcat90-mysql57-jdk8/20/0/playwright-report/data/43d173b462632127875ccf825fca4a24dab437e6.zip

**Comment by AliciaGarciaGarcia** [comment]
ci:test:playright

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:report:2549452062

**Comment by AliciaGarciaGarcia** [comment]
playwrigth test failing

**Comment by AliciaGarciaGarcia** [comment]
ci:test:playwright

**Comment by AliciaGarciaGarcia** [comment]
Playwright test finally passing!!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by robertoDiaz — APPROVED** [review]
Good to go!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
the use of `date-fns` was causing failures on other parts of playwright so using `moment`

**Comment by AliciaGarciaGarcia** [comment]
ci:test:playwright

**Comment by AliciaGarciaGarcia** [comment]
rebase needed

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
hey @ambrinchaudhary 

Can you recheck the changes on the tests?

Thanks!

**Review by ambrinchaudhary — APPROVED** [review]
LGTM!

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:reevaluate:1337401_5341

**Comment by AliciaGarciaGarcia** [comment]
failures not related.

**Comment by AliciaGarciaGarcia** [comment]
ci:forward:force

**Pull request closed without merging**